pkg/scheduling/requirements.go

```diff
@@ -219,5 +219,7 @@ func (r Requirements) Labels() map[string]string {
 
 func (r Requirements) String() string {
```

Comment:
sanity: How often is this method called? Is there any performance impact to doing sorting this method now?

Reply:
The function is called when we print the requirements. We only print out the requirements for the logging and events. So we can expect minimally performance impact